packages/node_modules/@ciscospark/widget-message-meet/src/container.js

```diff
@@ -219,7 +219,7 @@ export class MessageMeetWidget extends Component {
             }
             <div className={classNames(`ciscospark-activity-menu-button-wrapper`, styles.activityMenuButtonWrapper)}>
               <div className={classNames(`ciscospark-activity-menu-button`, styles.activityMenuButton)}>
-                <Button iconType={ICON_TYPE_WAFFLE} onClick={this.handleMenuClick} />
+                <Button accessibilityLabel={`Main Menu`} iconType={ICON_TYPE_WAFFLE} onClick={this.handleMenuClick} />
```

Comment:
The button gets its aria-label from the `accessibilityLabel` prop or just the `label` prop if there is one. Since all of our labels are capitalized, we should probably do it for the aria labels too. MDN has their example capitalized too: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute